packages/api/src/middleware/authentication.ts

```diff
@@ -228,9 +228,25 @@ const extractUserEmailFromAuthToken = async (
   next();
 };
 
+const extractUserIsAdminFromAuthToken = async (
+  req: Request<unknown, unknown, { admin?: boolean }>,
+  _: Response,
+  next: NextFunction
+): Promise<void> => {
+  const authenticationToken = getAuthTokenFromRequest(req);
+  const email = await getOptionalValueFromAuthToken(
```

Comment:
I found one more thing while testing locally - this function will throw an exception (because of the way the Fusionauth client works) if it receives an `authenticationToken` that's an empty string. Since we don't want that case to fail, we should check for an empty `authenticationToken` and then only call this if it's not empty.